core/src/main/scala/scalaz/Maybe.scala

```diff
@@ -5,6 +5,7 @@ import scala.util.control.NonFatal
 import scala.reflect.ClassTag
 import Ordering._
 import Isomorphism.{<~>, IsoFunctorTemplate}
+import scalaz.Maybe.{Empty, empty, just}
```

Comment:
it'd be best to refer to these with relative paths (which is idiomatic), e.g. `case Maybe.Empty() =>`